packages/api/server/http.mts

```diff
@@ -58,6 +58,32 @@ router.post('/disk', cors(), async (req, res) => {
   }
 });
 
+router.post(
+  '/file',
+  cors({
```

Comment:
Can we use cors here the way we used it for all the other functions? This doesn't work for me:

<img width="955" alt="Screenshot 2024-09-17 at 4 18 32 PM" src="https://github.com/user-attachments/assets/4b377196-c5a9-4eab-b96c-bbca77918ded">